doc/ctest_user_manual.md

```diff
@@ -94,7 +94,34 @@ When this is used, the check for allocations that have not been freed is checked
 This special fixture is executed before all the tests in the test suite. All resources allocated in `CTEST_SUITE_INITIALIZE` should be freed in `CTEST_SUITE_CLEANUP`.
 
 ```c
-CTEST_SUITE_INITIALIZE()
+#define CTEST_SUITE_INITIALIZE(funcName, ...) \
+    ...
+```
+
+The `...` argument is a list of 0 or more function names with the signature:
+
+```c
+void function(void)
+```
+
+These functions shall get called by the test runner before the `CTEST_SUITE_INITIALIZE` code is executed.
+
+Example usages:
+
+```c
+CTEST_SUITE_INITIALIZE(suite_init)
+{
+    /* Some init code */
+}
+```
+
+```c
```

Comment:
nit: can we put some text between these so it's clear that the first one is one example (no functions) and the second 2 blocks of code are another example (calling 1 function). Same below.

Reply:
Done.